docs/api-bridging/spec.md

```diff
@@ -0,0 +1,1193 @@
+
+This response gets repeated by the bridge in augmented form as:
+
+```JSON
+{
+    "type":  "findIntentResponse",
+    "payload": {
+        "intent":  "StartChat",
+        "appIntent":  {
+            "intent":  { "name": "StartChat", "displayName": "Chat" },
+            "apps": [
+                { "name": "Skype", "desktopAgent": "agent-B"},
+                { "name": "Symphony", "desktopAgent": "agent-B" },
+                { "name": "Symphony", "instanceId": "93d2fe3e-a66c-41e1-b80b-246b87120859", "desktopAgent": "agent-B" },
+                { "name": "Slack", "desktopAgent": "agent-B" }
+            ]
+        }
+    },
+    "meta": {
+        "requestGuid": "requestGuid",
+        "responseGuid":  "responseGuidAgentB",
+        "timestamp":  "2020-03-...",
+        "destination": {
+            "desktopAgent": "agent-A",
+            "name": "",
+            "appId": "",
+            "version": "",
+            // ... other metadata fields
+        },
+        "source": {
+            "desktopAgent": "agent-B",
+        }
+    }
+}
+```
+
+DA agent-C would produce response:
+
+```JSON
+{
+    "intent":  { "name": "StartChat", "displayName": "Chat" },
+    "apps": [
+       { "name": "WebIce"}
+    ]
+}
+```
+
+which is sent back over the bridge as a response to the request message as:
+
+```JSON
+// agent-C -> DAB
+{
+    "type":  "findIntentResponse",
+    "payload": {
+        "intent":  "StartChat",
+        "appIntent":  {
+            "intent":  { "name": "StartChat", "displayName": "Chat" },
+            "apps": [
+                { "name": "WebIce", "desktopAgent": "agent-C"}
+            ]
+        }
+    },
+    "meta": {
+        "requestGuid": "requestGuid",
+        "responseGuid":  "responseGuidAgentC",
+        "timestamp":  "2020-03-...",
+        "destination": {
+           "desktopAgent": "agent-A",
+           "name": "",
+           "appId": "",
+           "version": "",
+           // ... other metadata fields
+       }
+    }
+}
+```
+
+This response gets repeated by the bridge in augmented form as:
+
+```JSON
+{
+    "type":  "findIntentResponse",
+    "payload": {
+        "intent":  "StartChat",
+        "appIntent":  {
+            "intent":  { "name": "StartChat", "displayName": "Chat" },
+            "apps": [
+                { "name": "WebIce", "desktopAgent": "agent-C"}
+            ]
+        }
+    },
+    "meta": {
+        "requestGuid": "requestGuid",
+        "responseGuid":  "responseGuidAgentC",
+        "timestamp":  "2020-03-...",
+        "destination": {
+           "desktopAgent": "agent-A",
+           "name": "",
+           "appId": "",
+           "version": "",
+           // ... other metadata fields
+       },
+       "source": {
+            "desktopAgent": "agent-C",
+        }
+    }
+}
+```
+
+Then on agent-A the originating app finally gets back the following response from the bridge:
+
+```JSON
+// DAB -> agent-A

```

Comment:
Is this `DAB -> agent-A` or `agent-A -> app` ?

Does the DAB wait for responses from agents B and C then send the combined response to agent A, or does the DAB send the responses from B and C separately and agent A has to wait for all responses then combine?